pkg/store/copr/batch_coprocessor.go

```diff
@@ -206,7 +204,7 @@ func checkBatchCopTaskBalance(storeTasks map[uint64]*batchCopTask, balanceContin
 // balanceBatchCopTaskWithContinuity try to balance `continuous regions` between TiFlash Stores.
 // In fact, not absolutely continuous is required, regions' range are closed to store in a TiFlash segment is enough for internal read optimization.
 //
-// First, sort candidateRegionInfos by their key ranges.
+// First, the candidateRegionInfos is ordered for each partition. So there's no need to sort any more.
```

Comment:
```suggestion
// First, the caller should guarantee the candidateRegionInfos is ordered for each partition.
```

Reply:
If my understanding is correct, the region ranges from different partition will not overlap, in this case, we can make `candidateRegionInfos` be globally sorted instead of per partition sorted with a very small cost?